python/artm/hierarchy_utils.py

```diff
@@ -363,9 +363,9 @@ def save(self, path):
             level.save(os.path.join(path, "level" +
                                     str(level_idx) + "_pwt.model"), model_name="p_wt")
         info = {"parent_level_weight": [
-            level.phi_batch_weight for level in self._levels[1:]]}
+            level._phi_batch_weight for level in self._levels[1:]]}
         with open(os.path.join(path, "info.dump"), "wb") as fout:
-            pickle.dump(info, fout)
+            pickle.dump(info, fout, protocol=2)
```

Comment:
I think, for consistency with other code, we can just use `pickle` with default protocol version.